src/components/TeamMembers/TeamMembers.tsx

```diff
@@ -0,0 +1,240 @@
+import { useEffect, useState } from "react";
+import { BsTrash } from "react-icons/bs";
+import { FaUsers } from "react-icons/fa";
+import { HiOutlineMail } from "react-icons/hi";
+import {
+  addUserToTeam,
+  removeUserFromTeam,
+  getTeamMembers
+} from "../../api/services/teams";
+import { getPersons, User } from "../../api/services/users";
+import { Menu } from "../Menu";
+import { Avatar } from "../Avatar";
+import { IconButton } from "../IconButton";
+import { Modal } from "../Modal";
+import MultiSelectList from "../MultiSelectList/MultiSelectList";
+import { toastError, toastSuccess } from "../Toast/toast";
+import { useLoader } from "../../hooks";
+
+type TeamMembersProps = {
+  teamId: string;
+};
+
+export function TeamMembers({ teamId }: TeamMembersProps) {
+  const [teamMembers, setTeamMembers] = useState<User[]>([]);
+  const [allTeamMembers, setAllTeamMembers] = useState<User[]>([]);
+  const [selectedMembers, setSelectedMembers] = useState<User[]>([]);
+  const [open, setOpen] = useState<boolean>(false);
+  const { loading, setLoading } = useLoader();
+
+  async function fetchTeamMembers(id: string) {
+    setLoading(true);
+    const { data } = await getTeamMembers(id);
+    try {
+      const { data: users, error } = await getPersons();
+      setTeamMembers(data || []);
+      setAllTeamMembers(
+        (users || []).filter(
+          (user) => !data?.find((item) => item.id === user.id)
+        )
+      );
+      if (error) {
+        toastError(error.message);
+      }
+    } catch (ex) {
+      toastError(ex);
+    }
+    setLoading(false);
+  }
+
+  async function removeTeamMember(userId: string) {
+    try {
+      const { error } = await removeUserFromTeam(userId);
+      if (!error) {
+        toastSuccess("user removed from team successfully");
+        fetchTeamMembers(teamId);
+      } else {
+        toastError(error.message);
+      }
+    } catch (ex) {
+      toastError(ex);
+    }
+  }
+
+  async function addTeamMembers(selectedUsers: User[]) {
+    try {
+      const { error } = await addUserToTeam(
+        teamId,
+        selectedUsers.map((item) => item.id)
+      );
+      if (!error) {
+        toastSuccess("team members added successfully");
```

Comment:
Can we make this User.name added to team successfully ?

Reply:
we can also add multiple users at once, in that should I include all added user names in the toast message.